frontend/src/scenes/onboarding/sdks/sdk-install-instructions/js-web.tsx

```diff
@@ -19,7 +19,12 @@ export function JSSetupSnippet(): JSX.Element {
             {[
                 "import posthog from 'posthog-js'",
                 '',
-                `posthog.init('${currentTeam?.api_token}', { api_host: '${apiHostOrigin()}' })`,
+                `posthog.init('${currentTeam?.api_token}',`,
+                `    {`,
+                `        api_host: '${apiHostOrigin()}'`,
+                `        person_profiles: 'identified_only' // or 'always' to create profiles for anonymous users as well`,
```

Comment:
As a new user (and even a current one), its hard for me to know if I want to use 'identified_only' or 'always', so adding the comment here confuses me. Can we add a link to the docs in the comment where they can read more?

```suggestion
                `        person_profiles: 'identified_only' // or 'always' to create profiles for anonymous users as well`,
```

Reply:
I don't see a good way to add a docs link without confusing people more. The last thing I want to do is send people out of onboarding. I think people will largely stick to the default option set here.